app/assets/stylesheets/shared/inline_edit.scss

```diff
@@ -1,9 +1,9 @@
 // scss-lint:disable SelectorDepth
 // scss-lint:disable NestingDepth
+// scss-lint:disable SelectorFormat
+
 .sci-inline-edit {
-  column-gap: .25em;
-  display: grid;
-  grid-template-columns: auto repeat(2, max-content);
+  position: relative;
```

Comment:
What problem with grid? Why you need so many absolute? And duplicate code above?